pipelines/deploy-release/README.md

```diff
@@ -16,6 +16,10 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+### Changes in 0.14.0
+- modified the pipeline to dynamically source the `snapshot_spec.json` file
```

Comment:
Capital M to match the other entries please

Reply:
Fixed, thanks!